### PR TITLE
Inconsistency in the tabs shown to host and participants

### LINKED ISSUE #310: View All Submissions Tab for host redirects to Participant tab under
## I'm submitting a

- [x] Bug report

## Current behavior
Clicking on "view all submissions" redirects to "participate" Tab.

## Expected behavior
It should have open "view all submission" page instaed of redirecting to "participate" Tab

## Steps to reproduce
1. login as host
2. Go to hosted challenges -> click on hosted challenge -> click on "view all submission" tab

GIF
ezgif com-video-to-gif (5)

### PR DESCRIPTION
**Pull request by Kajol-Kumari**
Bug Fix [Inconsistency in the tabs shown to host and participants]

**Bug description**
The `view_all_submission` tab along with `close participation` button is visible to any normal user which should only be visible to the host of that challenge.

GIF of issue:
ezgif com-video-to-gif

GIF after Fix:
ezgif com-video-to-gif (1)

### CODE REVIEW

File: src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.ts [hunk 1]
```diff
@@ -181,7 +181,7 @@ export class ChallengeviewallsubmissionsComponent implements OnInit, AfterViewIn
     });
     this.challengeService.currentParticipationStatus.subscribe(status => {
       this.isParticipated = status;
-      if (!status) {
+      if (status) {
```
Comment (Shekharrajak) [inline src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.ts]: Then why do we need it here ?

**Comment by RishabhJain2018** [comment]
@lunayach Can you please test and review this?

**Review by Sanji515 — COMMENTED** [review]
@Kajol-Kumari It's still not working for seeded challenges.

**Comment by Kajol-Kumari** [comment]
> @Kajol-Kumari It's still not working for seeded challenges.

@Sanji515 can you please specify what's not working? As the the issue of `view_all_submission` tab has been already resolved after i added the participant privileges to the seeded challenge. I am updating this PR description. please take a look at it.

**Pull request merged**